serverScraper.js

```diff
@@ -23,6 +24,8 @@ setInterval(excecuteScraperGov, config.govInterval);
 setInterval(excecuteScraperVaccine, config.vaccineInterval);
 // Update vaccine data every  24 hours
 setInterval(excecuteScraperVaccineCoverage, config.vaccineCoverageInterval);
+// Update vaccine data every  24 hours
+setInterval(excecuteScraperVaccineStateCoverage, config.vaccineStateCoverageInterval);
```

Comment:
as I said, can just reuse this config variable here `vaccineCoverageInterval`

Reply:
done.